package.json

```diff
@@ -15,6 +15,7 @@
     "@types/ws": "8.5.4",
     "@typescript-eslint/eslint-plugin": "5.48.2",
     "@typescript-eslint/parser": "5.48.2",
+    "cross-spawn": "7.0.3",
```

Comment:
You moved this to dependencies, not devDependencies 🙃

Reply:
No?

Reply:
I was tired. I meant that you moved it to devDependencies, rather than removing it, and I didn't see any tests that used it.

Reply:
Yes, the point of the PR was to remove it from the actual package. The tests could be dealt with later (if at all, depending on what shenanigans I pulled there).